catroidTest/src/org/catrobat/catroid/uitest/web/ProjectUpAndDownloadTest.java

```diff
@@ -193,14 +187,7 @@ public void testUploadProjectOffensiveLanguageUsed() throws Throwable {
 
 		boolean uploadErrorOccurred = solo.waitForText(solo.getString(R.string.error_project_upload));
 
-		int statusCode = 0;
-		int statusCodeOffensiveLanguage = 511;
-		statusCode = (Integer) Reflection.getPrivateField(ServerCalls.getInstance(), "uploadStatusCode");
```

Comment:
Reflection removed, because I don't think a UI test should rely on a private field of a class it doesn't even know about.